scripts/config.py

```diff
@@ -98,8 +110,15 @@ def gather_data():
     api_key = prompt('Foxpass API Key')
     radius_secret = random_string(16)
 
-    public_ip = urlopen('http://169.254.169.254/latest/meta-data/public-ipv4').read()
-    private_ip = urlopen('http://169.254.169.254/latest/meta-data/local-ipv4').read()
+    if is_gce():
+        headers = {'Metadata-Flavor': 'Google'}
+        request = Request(METADATA_BASE_URL + 'computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip', headers=headers)
+        public_ip = urlopen(request).read()
+        request = Request(METADATA_BASE_URL + 'computeMetadata/v1/instance/network-interfaces/0/ip', headers=headers)
+        private_ip = urlopen(request).read()
+    else:
+        public_ip = urlopen('http://169.254.169.254/latest/meta-data/public-ipv4').read()
+        private_ip = urlopen('http://169.254.169.254/latest/meta-data/local-ipv4').read()
```

Comment:
No, not really. I was trying to not touch as much of the "original" stuff as possible. I can do back and update that though too if you want.